app/models/job.rb

```diff
@@ -4,4 +4,19 @@ class Job < ActiveRecord::Base
 
   has_many :tasks, dependent: :destroy
   has_many :awards, dependent: :destroy
+
+  before_destroy :remember_assignment
+  after_destroy :update_assignment
```

Comment:
I want to access assignment connected to just destroyed job in order to update homework.